packages/babel-plugin-proposal-private-methods/test/fixtures/accessors/tagged-template/exec.js

```diff
@@ -0,0 +1,16 @@
+class Foo {
```

Comment:
We don't need this test, since
```js
this.#tag``
```
is a private method and not an accessor (and we already test private methods in another field)